shuup/gdpr/resources.py

```diff
@@ -15,10 +15,28 @@
 from shuup.xtheme.resources import add_resource, InlineMarkupResource
 
 
-def add_gdpr_consent_resources(context, content):
-    # TODO: should we ignore admin urls?
+def valid_view(context):
     view_class = getattr(context["view"], "__class__", None) if context.get("view") else None
     if not view_class or not context.get("request"):
+        return False
+
+    request = context.get("request")
+    if request:
+        match = request.resolver_match
+        if match and match.app_name == "shuup_admin":
+            return False
+
+    view_name = getattr(view_class, "__name__", "")
+    if view_name == "EditorView":
+        return False
+
+    return True
+
+
+def add_gdpr_consent_resources(context, content):
+    # TODO: should we ignore admin urls?
```

Comment:
Not sure. Maybe remove this in case this is what you already. Commit message could be better.

Reply:
it's a leftover from old code (see line 19), wasn't planning on doing anything regarding this

Reply:
Ok based on the commit message and line 26 I expected this to actually ignore the admin URLs. Improve the commit message then.

Reply:
haha true :)